Added LambdaTest in the list of  cloud emulators (#1329)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

> If you are not able to use the emulators and do not have access to real devices, then cloud-based emulators are the next best thing.  A big advantage of cloud-based emulators over real devices and local emulators is that you are able to automate unit tests for your site across different platforms.  

Reviewer: probably should update the date at top of file.

*   [BrowserStack (commercial)][BrowserStack] is the easiest to use for manual testing.  You select an operating system, select your browser version and device type, select a URL to browse, and it spins up a hosted virtual machine with which you may interact.  You are able to also run multiple emulators in the same screen, enabling you to test the look and feel of your app across multiple devices at the same time.  

> *   [LambdaTest (commercial)][LambdaTest] will help you to perform manual cross browser testing on a combination of 2000+ browsers & operating systems. Users will be able to record video of complex bugs and even share it via integrations like MS Teams, Slack and more. Users can speed-up their testing by running tests in parallel.

Reviewer: ```suggestion
*   [LambdaTest (commercial)][LambdaTest] helps you perform manual cross-browser testing on a combination of 2000+ browsers and operating systems. Users can record videos of complex bugs and even share them through integration such as Microsoft Teams, Slack and more. Users can speed-up their testing by running tests in parallel.
```
An Edit pass.
